api/golang/core/lib/enclaves/enclave_context.go

```diff
@@ -345,6 +345,20 @@ func (enclaveCtx *EnclaveContext) DownloadFilesArtifact(ctx context.Context, art
 	return fileContent, nil
 }
 
+func (enclaveCtx *EnclaveContext) InspectFilesArtifact(ctx context.Context, artifactName services.FileArtifactName) (*kurtosis_core_rpc_api_bindings.InspectFilesArtifactContentsResponse, error) {
+	// TODO(vcolombo): Add a more intuitive return type to this call instead of returning the RPC response
+	response, err := enclaveCtx.client.InspectFilesArtifactContents(ctx, &kurtosis_core_rpc_api_bindings.InspectFilesArtifactContentsRequest{
```

Comment:
I think we could potentially return all the file-artifacts; but not urgent or needed today.